sdk/keyvault/keyvault-keys/test/internal/apiVersionParameter.spec.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import { createSandbox } from "sinon";
+import { KeyClient } from "../../src";
+import { ServiceClient } from "@azure/core-http";
+import { ClientSecretCredential } from "@azure/identity";
+import { env } from "@azure/test-utils-recorder";
+
+describe("The Keys client should set the apiVersion", () => {
+  const keyVaultUrl = `https://keyVaultName.vault.azure.net`;
+
+  it("it should default to the latest API version", async function() {
+    const sandbox = createSandbox();
+    const spy = sandbox.spy(ServiceClient.prototype, "sendRequest");
+
+    const credential = await new ClientSecretCredential(
+      env.AZURE_TENANT_ID!,
+      env.AZURE_CLIENT_ID!,
+      env.AZURE_CLIENT_SECRET!
+    );
+    const client = new KeyClient(keyVaultUrl, credential);
+    try {
+      await client.createKey("keyName", "RSA");
+    } catch {}
+
+    const calls = spy.getCalls();
+    assert.equal(
+      calls[0].args[0].url,
+      "https://keyVaultName.vault.azure.net/keys/keyName/create?api-version=7.1-preview"
```

Comment:
We have a constant somewhere that points to the default version right? We should use that here so that when the default version changes in the future we don't have to update this test.

Reply:
can do!